src/NzbDrone.Common/Processes/ProcessProvider.cs

```diff
@@ -320,7 +320,6 @@ private ProcessInfo ConvertToProcessInfo(Process process)
                 processInfo = new ProcessInfo();
                 processInfo.Id = process.Id;
                 processInfo.Name = process.ProcessName;
-                processInfo.StartPath = GetExeFileName(process);
```

Comment:
Do we still need to set StartPath here to `process.MainModule.FileName;`

Otherwise this change seems fine. Did it work for the Syno fix?